### ISSUE TITLE
Ability to pass job information to the Activator

### ISSUE THREAD

**Issue by youngcm2** [HangfireIO/Hangfire#198]
It would be nice if the JobActivator.ActivateJob received information about the job to allow the passing of parameters to the Kernel for Ninject.

```
public virtual object ActivateJob (Type jobType, ActivationContext context);

public class ActivationContext {
    public string JobId { get; private set; }
    public Job Job { get; private set; }
}
```

**Comment by Dresel** [comment]
What about setting a static scope for each job with an JobFilterAttribute (see [SO](http://stackoverflow.com/questions/27961210/hangfire-dependency-injection-lifetime-scope/28034217#28034217))?

**Comment by odinserj** [comment]
Closing this as already implemented in 1.6.0, full perform context will be available in 1.7.0.
